airbyte-cdk/python/airbyte_cdk/sources/streams/http/http.py

```diff
@@ -68,11 +68,9 @@ def clear_cache(self):
         """
         remove cache file only once
         """
-        STREAM_CACHE_FILES = globals().setdefault("STREAM_CACHE_FILES", set())
```

Comment:
@grubberr tagging you since you had the git blame: why did this block only allow a single removal of the cache file? the current change is passing unit tests. Is that an issue?

Reply:
@sherifnada oops I have answered on below